specification/trace/semantic_conventions/rpc.md

```diff
@@ -7,42 +7,56 @@ This document defines how to describe remote procedure calls
 
 <!-- toc -->
 
-- [gRPC](#grpc)
+- [Common remote procedure call conventions](#common-remote-procedure-call-conventions)
   * [Attributes](#attributes)
+- [gRPC](#grpc)
   * [Status](#status)
   * [Events](#events)
 
 <!-- tocstop -->
 
-## gRPC
+## Common remote procedure call conventions
 
-Implementations MUST create a span, when the gRPC call starts, one for
-client-side and one for server-side. Outgoing requests should be a span `kind`
-of `CLIENT` and incoming requests should be a span `kind` of `SERVER`.
+A remote procedure calls is described by two separate spans, one on the client-side and one on the server-side.
+For outgoing requests, the `SpanKind` MUST be set to `CLIENT` and for incoming requests to `SERVER`.
 
-Span `name` MUST be full gRPC method name formatted as:
+The _span name_ MUST be the full RPC method name formatted as:
 
 ```
 $package.$service/$method
 ```
 
-Examples of span name: `grpc.test.EchoService/Echo`.
+(where $service must not contain dots).
+
+Examples of span names:
+
+- `grpc.test.EchoService/Echo`
 
 ### Attributes
 
-| Attribute name | Notes and examples                                           | Required? |
-| -------------- | ------------------------------------------------------------ | --------- |
-| `rpc.service`  | The service name, must be equal to the $service part in the span name. | Yes |
-| `net.peer.ip`  | See [network attributes][]. | See below |
-| `net.peer.name`  | See [network attributes][]. | See below |
-| `net.peer.port`  | See [network attributes][]. | See below |
+| Attribute name |                          Notes and examples                            | Required? |
+| -------------- | ---------------------------------------------------------------------- | --------- |
+| `rpc.system`   | A string identifying the remoting system, e.g., `"grpc"` or `"java_rmi"`. | Yes       |
+| `rpc.service`  | The service name, must be equal to the $service part in the span name. | Yes       |
+| `rpc.method`   | The name of the method being called.                                   | Yes       |
+| `net.peer.ip`   | See [network attributes][]. | See below |
+| `net.peer.name` | See [network attributes][]. | See below |
+| `net.peer.port` | See [network attributes][]. | See below |
 
 At least one of [network attributes][] `net.peer.name` or `net.peer.ip` is required.
```

Comment:
What about "exotic" constellations like WCF using named pipe bindings? Change this to a SHOULD?

Reply:
The name of the named pipe would go into `net.peer.name` but without [`net.transport`](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/span-general.md#nettransport-attribute) it would hardly be helpful.
I could take this one over from messaging:
> Furthermore, it is strongly recommended to add the [`net.transport`][] attribute and follow its guidelines, especially for named pipe bindings.
[`net.transport`]: span-general.md#nettransport-attribute

Reply:
Mentioning net.transport sounds good. Actually, we could add a paragraph to the net attributes describing that convention. I imagine something like:

### Peer info

Some semantic conventions may say they require a "valid network peer info". This is defined as having either:
* At least one of `net.peer.ip` or `net.peer.name`
* `net.peer.transport` set to a non-IP based protocol, and `net.peer.name` set if applicable.

Reply:
let's do this in a follow-up PR and stick to RPC in this one 😉